src/app/data.rs

```diff
@@ -7,15 +7,15 @@ use wasm_bindgen::closure::Closure;
 
 type StoredPopstate = RefCell<Option<Closure<dyn FnMut(web_sys::Event)>>>;
 
-/// Used as part of an interior-mutability pattern, ie Rc<RefCell<>>
-#[allow(clippy::type_complexity, clippy::module_name_repetitions)]
-pub struct AppData<Ms: 'static, Mdl> {
+// Used as part of an interior-mutability pattern, ie Rc<RefCell<>>
+#[allow(clippy::type_complexity)]
+pub(crate) struct AppData<Ms: 'static, Mdl> {
     // Model is in a RefCell here so we can modify it in self.update().
     pub model: RefCell<Option<Mdl>>,
-    pub main_el_vdom: RefCell<Option<El<Ms>>>,
+    // @TODO_B: Document uses to look into removing/renaming
+    pub(crate) main_el_vdom: RefCell<Option<El<Ms>>>,
```

Comment:
Probably don't want to remove this one, since it's our reference for patch the rendered VDOM instead of recreating the VDOM in its entirety.